src/styles/resetForm.scss

```diff
@@ -0,0 +1,40 @@
+$primary: #0f0f0f;
+$baseColor: #232B32;
+$button: #A52A2A;
+$font: #ffffff;
+$neutralDark: #212121;
+$neutralPrimary: #3c3c3c;
+$neutralSecondary: #666666;
+$neutralQuaternary: #d0d0d0;
+
+@mixin flex {
```

Comment:
why do you place same mixin in multiple times?? read about scss imports. Pls learn scss docs!